locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -13,6 +13,17 @@ interface KeyToGrant {
   expiration?: number
 }
 export default class Dispatcher {
+  async getUser(network: number) {
```

Comment:
```suggestion
  async getPurchaser(network: number) {
```

^ Might be easier to understand what this is about.